src/utils/queryBuilder.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
+import moment from 'moment';
+
+// Builds up filter string by filter type
+const eventFilters = new Map([
+  ['startDate', val => `start_date gt ${moment(val).format('YYYY-MM-DD')}`],
+  ['searchText', val => `contains(title, '${val}') or contains(name, '${val}') or contains(description, '${val}')`]
```

Comment:
I can't find the documentation relating to `contains` in `https://opensupporter.github.io/osdi-docs/` and what is supported: case insensitive, exact phrase search, ect.